src/Http/Response.php

```diff
@@ -95,7 +95,16 @@ public static function makeFromExisting(IlluminateResponse $old)
      */
     public static function makeFromJson(JsonResponse $json)
     {
-        $new = static::create(json_decode($json->getContent(), true), $json->getStatusCode());
+        $content = $json->getContent();
+
+        // If the contents of the JsonResponse does not starts with /**/ (typical laravel jsonp response)
+        // we assume that it is a valid json response that can be decoded, or we just use the raw jsonp
+        // contents for building the response
+        if (! starts_with($json->getContent(), '/**/')) {
```

Comment:
Hum,

As we are in a static method and we do not instanciate the object we should create a `protected static function`. It looks dirty.

Should i extract-it into a static anyway ? Or should we merge this solution ?

Just tell me :)